### PR TITLE
Feat: Contact us modal

### PR DESCRIPTION
**Pull request by marwaghassa**
This resolve issue #32 
(Modal done without background design)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 import Navbar from "./components/Navbar/Navbar";
-
+import ContactUsModel from "./components/contactUsModel/contactUsModel";
 function App() {
+  const [open,setOpen]=useState(false)
   return (
     <Router>
       <Navbar />
+      <button className="text-xl text-white bg-red rounded-mg mx-auto pl-3 pr-3 ml-2" onClick={()=> setOpen(true)}> Keep in touch</button>
+     <ContactUsModel open={open} setOpen={setOpen} />
```
Comment (osamaakb) [inline src/App.js:13]: Give this code to @MiraMC149 , as she is the one will be calling the modal from her about page and remove it from here in the `app.js`

File: src/components/contactUsModel/contactUsModel.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import { Fragment, useRef, useState } from 'react'
+import { Dialog, Transition } from '@headlessui/react'
+
```
Comment (osamaakb) [inline src/components/contactUsModel/contactUsModel.jsx:3]: format the code in this file using `yarn format`

**Review by osamaakb — CHANGES_REQUESTED** [review]
Good job @marwaghassa , only few changes required

File: src/App.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import "./App.css";
-import React from "react";
+import React, { useState } from "react";
```
Comment (osamaakb) [inline src/App.js:2]: why you need `useState` here?

File: package.json [hunk 4]
```diff
@@ -3,14 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@afc-org/react-tailwind": "^0.1.1",
```
Comment (osamaakb) [inline package.json:6]: what is this library exactly do for you here? as I see you are using the modal from `headlessui` library

**Review by osamaakb — CHANGES_REQUESTED** [review]
please solve the conflicts + the comments below

**Comment by marwaghassa** [comment]
Screenshot (19)

**Review by osamaakb — COMMENTED** [review]
Good job @marwaghassa , I merged it from Mira's about us page PR, so I will close this

**Pull request closed without merging**